### PR TITLE
Fabo/fix proposal network switch issue

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f68cef73d363e00fbbbb4ad--lunieio.netlify.app

File: app/src/gql/index.js [hunk 1]
```diff
@@ -231,14 +231,6 @@ export const ProposalFragment = `
...
 `
 
-export const ProposalItem = (schema) => gql`
-  query proposal($id: String!) {
-    proposal(networkId: "${schema}", id: $id) {
-      ${ProposalFragment}
-    }
-  }
-`
-
```
Comment (Bitcoinera) [inline app/src/gql/index.js:241]: Oh, I see. Great!

**Comment by Bitcoinera** [comment]
I am still getting the same error:

image

**Comment by faboweb** [comment]
I can't reproduce. Then please take over

**Comment by Bitcoinera** [comment]
It seems changing the network triggers the proposal query for some reason...

**Comment by faboweb** [comment]
> It seems changing the network triggers the proposal query for some reason...

yes that is the issue

**Pull request closed without merging**